src/Components/Header/Header.js

```diff
@@ -42,7 +44,17 @@ export default function Header(){
             
             <NavLink to="/wishlist" >
                 <li>
-                    <i className="bi bi-heart"></i>
+                <div className="badge-wrapper">
+                    
+                    <i className="bi bi-heart fa-lg"></i>
+                    {isLogin ?
+                    <span className="icon-badge flex-align-center">
+                    {wishlist.length}</span>
+                    :
+                    <span></span> }
```

Comment:
Instead of empty span tags, you can return null. Also you can simply use **&&** in order to avoid ternary operators in such situations